tests/unit/controller/eme-controller.ts

```diff
@@ -295,18 +295,7 @@ describe('EMEController', function () {
 
     media.emit('encrypted', badData);
 
-    expect(emeController.keyIdToKeySessionPromise.f000ba00).to.be.a('Promise');
-    if (!emeController.keyIdToKeySessionPromise.f000ba00) {
-      return;
-    }
-    return emeController.keyIdToKeySessionPromise.f000ba00
-      .catch(() => {})
-      .finally(() => {
-        expect(emeController.hls.trigger).callCount(1);
-        expect(emeController.hls.trigger.args[0][1].details).to.equal(
-          ErrorDetails.KEY_SYSTEM_NO_SESSION,
-        );
-      });
+    expect(emeController.keyIdToKeySessionPromise).to.deep.equal({});
```

Comment:
suggestion(non-blocking): nitpick - This assertion isn't very intuitive. Consider adding a more descriptive message associated with the expectation, a la:
```suggestion
    expect(emeController.keyIdToKeySessionPromise).to.deep.equal({}, 'should be an empty dictionary when no encyprtion key ids are found in the media');
```

Reply:
done